uspace/Makefile.common

```diff
@@ -231,6 +235,14 @@ HELENOS_CFLAGS = \
 # CFLAGS variable is traditionally used for overridable flags.
 CFLAGS = $(COMMON_CPPFLAGS) $(COMMON_CFLAGS) $(HELENOS_CFLAGS) $(DEFAULT_CFLAGS)
 
+# Flags for the compilation of C++ code.
+CPPFLAGS = $(LIBCPP_INCLUDES_FLAGS) $(INCLUDES_FLAGS) -O$(OPTIMIZATION) -imacros $(CONFIG_HEADER) \
```

Comment:
Should be CXXFLAGS. CPPFLAGS is usually used for C preprocessor.

Reply:
Fixed, all occurences of CPP (except for LIBCPP) were changed to CXX in the build system.